moped-editor/src/views/projects/projectView/ProjectPhase/helpers.js

```diff
@@ -0,0 +1,183 @@
+import { useMemo } from "react";
+import * as yup from "yup";
+
+export const phaseValidationSchema = yup.object().shape({
+  phase_id: yup
+    .number("Phase is required")
+    .nullable()
+    .required("Phase is required"),
+  subphase_id: yup.number().nullable().optional(),
+  phase_start: yup
+    .date()
+    .nullable()
+    .optional()
+    .when("is_current_phase", {
+      is: true,
+      then: (schema) =>
+        schema.required("Start date is required when phase is current"),
+    })
+    .typeError("Invalid Date"),
+  phase_end: yup.date().nullable().optional().typeError("Invalid Date"),
+  is_current_phase: yup.boolean(),
+  is_phase_start_confirmed: yup.boolean(),
+  is_phase_end_confirmed: yup.boolean(),
+  description: yup
```

Comment:
**Change request** I just tested this rule in the form, and I think this key needs to be updated to `phase_description`. I was able to submit more than 500 chars and couldn't get the error message to appear.

Reply:
nice catch! 🙏